### DISCUSSION TITLE
velero version doesn't get server version

### DISCUSSION THREAD

**Discussion by koehn** [vmware-tanzu/velero#3336] [Community support Q&A] [answered]
---
name: velero version doesn't get server version
about: When I run `velero version` the server version is never displayed

---

**What steps did you take and what happened:**
```
$ velero version
Client:
	Version: v1.5.2
	Git commit: -
Server:
	Version: 
$ 
```

**What did you expect to happen:**
Expect to see the client and server version. 

**The output of the following commands will help us better understand what's going on**:
(Pasting long output into a [GitHub gist](https://gist.github.com) or other pastebin is fine.)

- `kubectl logs deployment/velero -n velero`
```
time="2021-01-15T22:03:32Z" level=info msg="Processing new ServerStatusRequest" controller=serverstatusrequest logSource="pkg/controller/server_status_request_controller.go:86" phase= serverStatusRequest=velero/velero-cli-ndhnc
time="2021-01-15T22:03:32Z" level=error msg="Unable to update the request" controller=serverstatusrequest error="serverstatusrequests.velero.io \"velero-cli-ndhnc\" not found" error.file="/go/src/github.com/vmware-tanzu/velero/internal/velero/serverstatusrequest.go:49" error.function="github.com/vmware-tanzu/velero/internal/velero.(*ServerStatus).PatchStatusProcessed" logSource="pkg/controller/server_status_request_controller.go:89" phase= serverStatusRequest=velero/velero-cli-ndhnc
```

**Anything else you would like to add:**
I'm running velero 1.5.3 on the server. 

**Environment:**

- Velero version (use `velero version`):v1.5.2 - v1.5.3 (server)
- Velero features (use `velero client config get features`): `features: <NOT SET>`
- Kubernetes version (use `kubectl version`):
```
Client Version: version.Info{Major:"1", Minor:"20", GitVersion:"v1.20.2", GitCommit:"faecb196815e248d3ecfb03c680a4507229c2a56", GitTreeState:"clean", BuildDate:"2021-01-14T05:15:04Z", GoVersion:"go1.15.6", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"20", GitVersion:"v1.20.1", GitCommit:"c4d752765b3bbac2237bf87cf0b1c2e307844666", GitTreeState:"clean", BuildDate:"2020-12-18T12:00:47Z", GoVersion:"go1.15.5", Compiler:"gc", Platform:"linux/amd64"}
```

- Kubernetes installer & version:
- Cloud provider or hardware configuration: hetzner, 5 2CPU/8GB RAM worker nodes
- OS (e.g. from `/etc/os-release`): - RuntimeOS: darwin - RuntimeArch: amd64

**Vote on this issue!**

This is an invitation to the Velero community to vote on issues, you can see the project's [top voted issues listed here](https://github.com/vmware-tanzu/velero/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc).  
Use the "reaction smiley face" up to the right of this comment to vote.

- :+1: for "I would like to see this bug fixed as soon as possible"
- :-1: for "There are more important bugs to focus on right now"

**Comment by zubron** [comment, marked as answer]
Hi @koehn! Is this a new installation of Velero, or did you perform an upgrade from a previous version (1.4 or earlier)? This looks like it may be caused by having older versions of the CRDs installed in the cluster.

If this is an upgrade from 1.4 or earlier, please review our upgrade instructions: https://velero.io/docs/v1.5/upgrade-to-1.5/ and follow the steps to upgrade the versions of the CRDs in the cluster.

If this is a new installation, could you provide the command used to install? Thanks!
